### ISSUE TITLE
`cargo test` is failing with last nightly

### ISSUE THREAD

**Issue by calixteman** [getsentry/symbolic#276]
```
test test_elf_functions ... FAILED
test test_mach_functions ... FAILED
test test_elf_files ... FAILED
test test_elf_symbols ... FAILED
test test_mach_files ... FAILED
test test_mach_symbols ... FAILED
test test_pdb_symbols ... FAILED
```
The error is always the same:
```
thread 'test_elf_functions' panicked at 'attempted to leave type `base::Symbol` uninitialized, which is invalid', /home/calixte/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/mod.rs:658:9
```

**Comment by jan-auer** [comment]
Thanks for reporting, @calixteman. This panic is occurring in `dmsort`. An issue has already been raised in https://github.com/emilk/drop-merge-sort/issues/2.

**Comment by calixteman** [comment]
The `dmsort` doesn't seem be maintained anymore...
Could you think about a replacement ?

**Comment by jan-auer** [comment]
I'll look into this. We chose dmsort specifically as it was significantly faster than using the standard sort function, so we'll have to find a suitable replacement.

**Comment by jan-auer** [comment]
Should be fixed via #287

**Comment by Swatinem** [comment]
*nightly* from oktober 2nd is stable now, so closing this.

We do have a CI job thats running on nightly, but that is blocked by new clippy lints as it seems.
